autopush/router/simple.py

```diff
@@ -107,8 +108,8 @@ def route_notification(self, notification, uaid_data):
             if result is False:
                 self.metrics.increment("router.broadcast.miss")
                 returnValue(self.stored_response(notification))
-        except ProvisionedThroughputExceededException:
-            raise RouterException("Provisioned throughput error",
+        except (ProvisionedThroughputExceededException, JSONResponseError):
```

Comment:
turns out ProvisionedThroughputExceededException is a subclass of JSONResponseError: so you could just catch that one